spyder/plugins/variableexplorer/widgets/collectionsdelegate.py

```diff
@@ -112,6 +113,32 @@ def createEditor(self, parent, option, index, object_explorer=False):
             value = self.get_value(index)
             if value is None:
                 return None
+        except ImportError as msg:
+            self.sig_editor_shown.emit()
+            module = str(msg).split("'")[1]
+            if module in ['pandas', 'numpy']:
+                if module == 'numpy':
+                    val_type = 'array'
+                else:
+                    val_type = 'dataframe or series'
+                QMessageBox.critical(
+                    self.parent(), _("Error"),
+                    _("Spyder is unable to show the {val_type} you're "
+                      "trying to visualize because <i>{module}</i> was "
```

Comment:
What do you think about using a monospaced font for `module` instead of italics?

I mean, things in the dialog would like

> Spyder is unable to show the array you're trying to visualize because `numpy` was not ...

instead of

> Spyder is unable to show the array you're trying to visualize because *numpy* was not ...